src/content/whats-new/2024/02/whats-new-02-16-infra-otel-guids.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Changes to host GUIDs for hosts instrumented with OpenTelemetry"
```

Comment:
I'd emphasise it's the entity.guid that's changing, maybe "Changes to entity GUIDs for hosts instrumented with OpenTelemetry"

Reply:
Updated in a recent commit